CMakePresets.json

```diff
@@ -0,0 +1,134 @@
+{
+  "version": 3,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 20,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "community",
+      "displayName": "Build ArangoDB Community Edition",
+      "binaryDir": "${sourceDir}/build/${presetName}",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
+        "USE_ENTERPRISE": "Off",
+        "USE_JEMALLOC": "On",
+        "USE_IPO": "On"
+      }
+    },
+    {
+      "name": "enterprise",
+      "inherits": "community",
+      "displayName": "Build ArangoDB Enterprise Edition",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
+        "USE_ENTERPRISE": "On",
+        "USE_JEMALLOC": "On",
+        "USE_IPO": "On"
+      }
+    },
+    {
+      "name": "community-developer",
+      "inherits": "community",
+      "displayName": "Build ArangoDB Community Edition (Developer Build)",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
```

Comment:
Debug. RelWithDebInfo build long and useless

Reply:
I have no real preference, I have always used RelWithDebInfo in the past.
What is the benefit of Debug?

Reply:
> What is the benefit of Debug?

Faster to build, better stacks